src/KeyringController.test.ts

```diff
@@ -65,56 +72,59 @@ describe('KeyringController', function () {
       expect(keyringController.keyrings).toHaveLength(0);
     });
 
-    it('emits "lock" event', async function () {
-      const spy = sinon.spy();
-      keyringController.on('lock', spy);
+    it('emits "lock" event', async () => {
+      const lockSpy = spy();
+      keyringController.on('lock', lockSpy);
 
       await keyringController.setLocked();
 
-      expect(spy.calledOnce).toBe(true);
+      expect(lockSpy.calledOnce).toBe(true);
     });
   });
 
-  describe('submitPassword', function () {
-    it('should not load keyrings when incorrect password', async function () {
+  describe('submitPassword', () => {
+    it('should not load keyrings when incorrect password', async () => {
       await keyringController.createNewVaultAndKeychain(PASSWORD);
       await keyringController.persistAllKeyrings();
       expect(keyringController.keyrings).toHaveLength(1);
 
       await keyringController.setLocked();
 
       await expect(
-        keyringController.submitPassword(`${PASSWORD}a`),
+        keyringController.submitPassword(`${PASSWORD as string}a`),
```

Comment:
Any way to avoid this typecast? Seems a bit odd.

Reply:
[91fd437](https://github.com/MetaMask/KeyringController/pull/202/commits/91fd4378dfc92de33b9b8203b39845157eac9244)